eve/workers/unit_and_feature_tests/Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM spotify/kafka
+
+#
+# Install apt packages needed by backbeat and buildbot_worker
+#
+COPY ./backbeat_packages.list ./buildbot_worker_packages.list /tmp/
+
+RUN curl -sL https://deb.nodesource.com/setup_6.x | bash - \
+    && cat /tmp/*packages.list | xargs apt-get install -y \
+    && pip install pip==9.0.1 \
+    && rm -rf /var/lib/apt/lists/* \
+    && rm -f /tmp/packages.list
+
+#
+# Add user eve
+#
+RUN adduser -u 1042 --home /home/eve --disabled-password --gecos "" eve \
+    && adduser eve sudo \
+    && sed -ri 's/(%sudo.*)ALL$/\1NOPASSWD:ALL/' /etc/sudoers
+
+#
+# Run buildbot-worker on startup through supervisor
+#
+ARG BUILDBOT_VERSION=0.9.1
+
+RUN pip install buildbot-worker==$BUILDBOT_VERSION
+ADD supervisor/buildbot_worker.conf /etc/supervisor/conf.d/
+
+#
+# Configure build env
+#
+ENV CXX=g++-4.9
```

Comment:
Why is it needed to specify the version of gcc? If we can avoid this it would ease maintenance since eventually we will want to use a more recent gcc.

Reply:
We can change it to a newer version or eliminate it all together but the goal was for consistency during testing.

Reply:
If it works without, I would personally rather remove it, better use the system compiler version I think unless it's shown to cause issues.